app/src/main/AndroidManifest.xml

```diff
@@ -281,6 +281,10 @@
         <meta-data
             android:name="firebase_analytics_collection_enabled"
             android:value="false" />
+        <provider
```

Comment:
Might be worthwhile to add a comment here, its not immediately clear at a glance that this removes the work manager initializer (even though this is what the docs recommend)

Reply:
yh for sure. @mcomella  quickly wanted to see the changes for #11698  (did some performance test for the workmanager). Im not sure if we are landing this yet.